packages/webservice/app/src/main/resources/application.yaml

```diff
@@ -0,0 +1,39 @@
+elide:
+  modelPackage: 'com.yahoo.navi.ws.models.beans'
+  json-api:
+    path: /api/v1
+    enabled: true
+  graphql:
+    path: /graphql/api/v1
+    enabled: false
+  swagger:
+    path: /doc
+    enabled: true
+  async:
+    enabled: false
```

Comment:
Don't you want async turned on?

Reply:
I was trying to find a why to get rid of the analytic models in swagger, this didn't change anything. I can clean this up.